primehub/cli.py

```diff
@@ -274,7 +274,8 @@ def main(sdk=None):
             print(f'unknown API errors {e}', file=sdk.stderr)
             sys.exit(1)
         errors = e.args[0]['errors']
-        sdk.config.get_display().display(None, errors, sdk.stderr)
+        printer = sdk.commands.items()[0][1]
```

Comment:
It is confusing to me. It could be clearer by comment

```python
# We need a display to show error, but a display lives with a command group
# Get one from config command (it could be from any command)
sdk.config.get_display().display(None, errors, sdk.stderr)
```

Reply:
yeah, not so intuitive. I added this as we have a regular-command pool and an admin-command pool now.
I will add a comment for it.